feature/chat/src/main/java/com/napzak/market/chat/chatlist/ChatListViewModel.kt

```
@@ -1,20 +1,74 @@
+            } catch (e: IllegalStateException) {
+                Timber.e(e)
+            }
+        }
+    }
+
+    private fun updateChatRoom(message: ReceiveMessage<*>) {
+        _chatRoomsState.update { currentState ->
+            try {
+                if (currentState is UiState.Success) {
+                    val chatRooms = currentState.data.toMutableMap()
+                    val roomId = message.roomId ?: throw Exception()
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)